tests/basic/storage_pool_test.go

```diff
@@ -5188,3 +5190,195 @@ var _ = Describe("{ResizePoolDrivesInDifferentSize}", func() {
+	// 3) Update the label for the pool before expand
+	// 4) perform volume delete
+	// 5) Expand by resize the pool when delete is in progress
+	// 6) Check the alert for the pool expand
+	// 7) check the labels after pool expand
+	var testrailID = 51285
+	// testrailID corresponds to: https://portworx.testrail.net/index.php?/tests/view/51285
+
+	var runID int
+	JustBeforeEach(func() {
+
+		StartTorpedoTest("VolDeletePoolExpand", "Delete volume which has ~200G data and do an expansion of pool by resize", nil, testrailID)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+
+	})
+	var contexts []*scheduler.Context
+	var newContexts []*scheduler.Context
+
+	stepLog := "should get the existing storage node and write ~200G data to a volume"
+
+	It(stepLog, func() {
+
+		log.InfoD(stepLog)
+		contexts = make([]*scheduler.Context, 0)
+
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("voldeletepoolexpand-%d", i))...)
+		}
+
+		ValidateApplications(contexts)
+
+		log.Infof("Need to check if volume is close to 200G occupied")
+		vol, err := getVolumeWithMinimumSize(contexts, 90)
+
+		// We will change the size, after modifying/deploying a vdbench/fio to write ~200G. Current vdbench is writing 98G
+		dash.VerifyFatal(err, nil, "Checking if the desired volume is obtained")
+		volID := vol.ID
+		volName := vol.Name
+
+		log.Infof("The volume that is having size used around 190 G is %s with name %s", volID, volName)
+
+		var poolIDToResize string
+		pools, err := Inst().V.ListStoragePools(metav1.LabelSelector{})
+		log.FailOnError(err, "Failed to list storage pools")
+		dash.VerifyFatal(len(pools) > 0, true, " Storage pools exist?")
+
+		// Pick a pool from a pools list and resize it
+		appVol, err := Inst().V.InspectVolume(volID)
+		dash.VerifyFatal(err, nil, "Checking if the Volume inspect is success for the desired volume")
+		// Get the pool UUID on which the volume which is ~190G exist
+		poolIDToResize = appVol.ReplicaSets[0].PoolUuids[0]
+		log.FailOnError(err, "error identifying pool to run test")
+		dash.VerifyFatal(len(poolIDToResize) > 0, true, fmt.Sprintf("Expected poolIDToResize to not be empty, pool id to resize %s", poolIDToResize))
+		poolToBeResized := pools[poolIDToResize]
+		dash.VerifyFatal(poolToBeResized != nil, true, "Pool to be resized exist?")
+
+		// Px will put a new request in a queue, but in this case we can't calculate the expected size,
+		// So need to wait until the ongoing operation is completed
+		time.Sleep(time.Second * 60)
+		stepLog = "Verify that pool resize is not in progress"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			if poolResizeIsInProgress(poolToBeResized) {
+				// Wait until resize is completed and get the updated pool again
+				poolToBeResized, err = GetStoragePoolByUUID(poolIDToResize)
+				log.FailOnError(err, "Failed to get pool using UUID %v", poolToBeResized)
```

Comment:
Can we be consistent and change it to a string formatter `%s`, I see you used it correctly on line 5335, we know this is a `string`

Reply:
done